dtfs-central-api/src/helpers/isNumber.test.js

```diff
@@ -0,0 +1,58 @@
+const isNumber = require('./isNumber');
+
+describe('isNumber', () => {
+  // Tests that the function correctly identifies a single digit number
+  it('should correctly identify a single digit number', () => {
```

Comment:
I don't think you need the comments, I think your test names do a good job of explaining what the test does :)

Reply:
Have updated the documentation.